### ISSUE TITLE
Let Joker become Crewmate

### ISSUE THREAD

**Issue by ChrissW-R1** [NotHunter101/ExtraRolesAmongUs#52]
If only three persons left in the game and the joker is still alive it is a bad situation. Mostly two persons are suspicious, the impostor and the joker.

So this is, what the people knew:
The impostor and the joker knew all roles of the left players. Only the last crewmate know its own role and that there an impostor and a joker in the group.

Desires of the players are:
- Impostor: Have to kill one more player
- Joker: Have to voted
- Crewmate: Have to vote the impostor

The game can now end in the following ways:
- Joker can vote himself, but have only a 50% chance, that also the crewmate votes him, too. Otherwise he will lose the game.
- Impostor have to vote the crewmate or skip to win.
All other ways are not useful, if everyone want to win. So the crewmate have no chance to win. So the crewmate have no chance to win and the impostor and the joker each have a 50% chance to win. This is only luck and there is no tactics, which is one of the basics in this game.

If the joker becomes a real impostor, the impostors win instantly. So he have to become a crewmate. Now the game is more like the basic game and both remaining fractions have a good chance to win with skill.

**Comment by benniii1337** [comment]
That's indeed a big problem with the Joker. Before playing this mod myself, i did saw it in twitch streams, where the group decided that the joker should be a crewmate in this situation, as you said. 

What my group actually does, and this i what i like in this mod among the others most: Not playing with 100% chance for the extra roles. That way you don't actually know if the joker is even in the game. So if we come across the 1vs1vs1 situation (which is not often, because the joker could be killed from imp/sheriff or isn't in the game), we just play it out. 

But im open for changes. 

The Joker could have an extra button like the Engineer, which he can only press when there are 3 alive to transform himself to the crewmate role or smth like that. Maybe he have even to do 1 task then.

**Comment by ChrissW-R1** [comment]
No, also with a non-100%-chance for this role. The situation is the same. If there are only three persons and it looks like there are two impostors alive, a joker have to be alive and in the game or you have a troll in your group. -.-

**Comment by NotHunter101** [comment]
An idea has been proposed by some other people in the Discord that I really liked; once the taskbar fills to a certain configurable amount (i.e 70%), the Joker will have one more chance to get voted off before he dies (by exile, so there is no corpse) at the end of the meeting. This would also encourage Crewmates to complete their tasks quickly, which also gives the Impostors an extra edge (which they certainly need in the current balance).

**Comment by NotHunter101** [comment]
For now, I'll leave this issue open as a pseudo-suggestion board so people who might have a better idea of how to balance the Joker can leave their thoughts.

**Comment by ChrissW-R1** [comment]
Other solutions of this problem are also a good idea. I'm open for suggestions.
That the joker will become is crewmate is only one possible way to solve this situation.
